README.md

```diff
@@ -88,18 +138,28 @@ L.tileLayer('https://{s}.tiles.mapbox.com/v3/anishshah101.ipm9j6em/{z}/{x}/{y}.p
   id: 'examples.map-i86knfo3'
 }).addTo(map);
 
+// keymap guide
+var keymapper = {
+  position: "topright",

```

Comment:
Is it possible to provide a `popup`-style toolbar just like the default, but with different tools in it, and how would we describe the `position` if we did that?

Reply:
@jywarren, are you suggesting more than one popup toolbars associated with one overlay? I believe that can be followed up in a new issue on top of this one. As far as this goes, I've used `L.DomUtil.create` method at its core to create custom **static** toolbars on different parts of the view. What do you think?

Reply:
> Is it possible to provide a popup-style toolbar just like the default, but with different tools in it, and how would we describe the position if we did that?

This is absolutely possible, although I doubt the rigidity of the final product since Leaflet doesn't seem to natively support positioning multiple popups in a user-defined orientation. Nonetheless, I'm willing to give this a try and construct this from scratch if that's what's required here. This is interesting! :smile:

Thanks!!

Reply:
https://github.com/publiclab/Leaflet.DistortableImage/issues/140#issuecomment-473101722

Reply:
Migrating this over to #170.

Reply:
> more than one popup toolbars associated with one overlay

Sorry, no, just meant the ability to supply a different interface (like a popup toolbar) that would appear upon clicking the image. Hopefully this addresses your "rigidity" comment as well? But yes, and you sum things up nicely in #170. 

Great.